# HG changeset patch
# User Francois Bissey <francois.bissey@canterbury.ac.nz>
# Date 1360534690 -46800
# Node ID 792a893b7ff2a0e57b4c3e3e7106e6f660344d90
# Parent  ca877a1fdc06f49f74e422f648ed5c05fea1a570
Cut the stuff we don't need for checking the gap install. It furthermore would make our life difficult (lib/lib64).

diff --git a/sage/libs/gap/util.pyx b/sage/libs/gap/util.pyx
--- a/sage/libs/gap/util.pyx
+++ b/sage/libs/gap/util.pyx
@@ -157,14 +157,9 @@
         '/home/vbraun/opt/sage-5.3.rc0/local/gap/latest'
     """
     import os.path
-    gapdir = os.path.join(SAGE_LOCAL, 'gap', 'latest')
-    if os.path.exists(gapdir):
-        return gapdir
-    print 'The gap-4.5.5.spkg (or later) seems to be not installed!'
     gap_sh = open(os.path.join(SAGE_LOCAL, 'bin', 'gap')).read().splitlines()
     gapdir = filter(lambda dir:dir.strip().startswith('GAP_DIR'), gap_sh)[0]
     gapdir = gapdir.split('"')[1]
-    gapdir = gapdir.replace('$SAGE_ROOT', SAGE_ROOT)
     return gapdir
    
 
